home/web/client/components/AccountsDetails/AccountsDetails.js

```diff
@@ -1,40 +1,55 @@
-import React from 'react';
+import React, { Component, PropTypes } from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import { isEqual } from 'lodash';
 
 import FlatButton from 'material-ui/FlatButton';
 import TextField from 'material-ui/TextField';
 import Dialog from 'material-ui/Dialog';
 import AddIcon from 'material-ui/svg-icons/content/add';
 
-import Web3Component from '../Web3Component';
-import Identicon from '../Identicon';
+import Identicon from 'dapps-react-components/src/Identicon';
 
 import resetStyles from '../../reset.css';
 import styles from './AccountsDetails.css';
 
-export default class AccountDetails extends Web3Component {
+import { updateAccountsNames } from '../../actions/rpc';
 
-  // IE9 - contextTypes are not inherited
-  static contextTypes = Web3Component.contextTypes;
+export default class AccountsDetails extends Component {
 
-  state = {};
+  static contextTypes = {
```

Comment:
line 77
`const address = this.context.web3.toChecksumAddress(acc)`

will b removed in future PR